src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,306 @@
+  showSocialIcons = true,
+  showLegalText = true,
+  googleLoginUrl = 'https://accounts.google.com/signin',
+  githubLoginUrl = 'https://github.com/login',
+  facebookLoginUrl = 'https://www.facebook.com/login',
+  privacyPolicyUrl = 'https://policies.google.com/privacy',
+  termsOfServiceUrl = 'https://policies.google.com/terms',
+}) => {
+  const [emailOrUsername, setEmailOrUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const [errors, setErrors] = useState<{ emailOrUsername: string; password: string }>({
+    emailOrUsername: '',
+    password: '',
+  });
+
+  const theme = useTheme();
+
+  const handleImgError = (e: React.SyntheticEvent<HTMLImageElement>) => {
+    const img = e.currentTarget;
+    const alt = img.alt;
+    if (alt === 'Google' && img.src !== GoogleIcon) {
+      img.src = GoogleIcon ?? '';
+    } else if (alt === 'GitHub' && img.src !== GithubIcon) {
+      img.src = GithubIcon ?? '';
+    } else if (alt === 'Facebook' && img.src !== FacebookIcon) {
+      img.src = FacebookIcon ?? '';
+    }
+  };
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+
+    const newErrors = { emailOrUsername: '', password: '' };
+
+    if (!emailOrUsername) {
+      newErrors.emailOrUsername = 'Email or Username is required.';
+    } else if (emailOrUsername.includes('@') && !EMAIL_REGEX.test(emailOrUsername)) {
+      newErrors.emailOrUsername = 'Please enter a valid email address.';
+    }
+
+    if (!password) {
+      newErrors.password = 'Password is required.';
+    } else if (password.length < PASSWORD_MIN_LENGTH) {
+      newErrors.password = `Password must be at least ${PASSWORD_MIN_LENGTH} characters long.`;
+    }
+
+    if (newErrors.emailOrUsername || newErrors.password) {
+      setErrors(newErrors);
+      return;
+    }
+
+    setErrors(newErrors);
+    onSubmit({ emailOrUsername, password });
+  };
+
+  const isDisabled = disabled || loading || !!errors.emailOrUsername || !!errors.password;
+
+  return (
+    <Box sx={getPageContainerStyles()}>
+      <Box component="form" onSubmit={handleSubmit} sx={getFormStyles(theme)}>
+        <Box sx={getLogoStyles(theme)}>
+          <Logo orientation="horizontal" width="100%" height="100%" alt="Logo" />
+        </Box>
+
+        <FormInput
+          label="Mail Id"
+          value={emailOrUsername}
+          onChange={(e) => {
+            setEmailOrUsername(e.target.value);
+            setErrors({ ...errors, emailOrUsername: '' });
+          }}
+          placeholder="Mail Id"
+          errorMessage={errors.emailOrUsername}
+          sx={getInputStyles(GEOMETRY_PX.EMAIL_USERNAME_INPUT_TOP, theme)}
+        />
+        {errors.emailOrUsername && (
+          <Typography
+            variant="caption"
+            sx={getErrorTextStyles(GEOMETRY_PX.EMAIL_USERNAME_ERROR_TOP, theme)}
+          >
+            {errors.emailOrUsername}
+          </Typography>
+        )}
+
+        <FormInput
```

Comment:
Password input is also missing an explicit accessible label and name/id. Inputs must have labels for screen readers and keyboard users.

**Sugerencia:**
```typescript
Add id/name/aria-label and render an associated <label> or use aria-label/aria-labelledby. Also ensure password input has autocomplete="current-password".
```